Sources/Data/models/Model.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Model15.xcdatamodel</string>
+	<string>Model16.xcdatamodel</string>
```

Comment:
I do not think migration is needed when adding a new attribute. Or do you update existing ones too?

Reply:
Good catch. Was debating changing the existed one at the beginning but ended up with no change. Will revert this change.

Reply:
resolve this comment as its been fixed in [c5f423c](https://github.com/brave/brave-ios/pull/5910/commits/c5f423ca50c586bafae77e71e382a480eb2d05e5)